lib/nrf_modem_lib/nrf_modem_lib.c

```diff
@@ -55,6 +59,42 @@ static const nrf_modem_init_params_t init_params = {
 #endif
 };
 
+static void log_fw_version_uuid(void)
+{
+	int err;
+	char fw_version_buf[32];
+	char *fw_version;
+	char fw_uuid_buf[64];
+	char *fw_uuid;
+
+	err = nrf_modem_at_cmd(fw_version_buf, sizeof(fw_version_buf), "AT+CGMR");
```

Comment:
Can we use `nrf_modem_at_scanf` instead and avoid bringing in `strtok`

Reply:
actually, let's stick to `_at_cmd` which has lower stack requirements, but let's use `strstr` instead of `strtok`?
We don't need to tokenize the string after all.

Reply:
Is there any particular reason to prefer `strstr` to `strtok`?